src/auth/session/storage/redis.ts

```diff
@@ -71,6 +71,32 @@ export class RedisSessionStorage implements SessionStorage {
     return true;
   }
 
+  public async deleteSessions(ids: string[]): Promise<boolean> {
+    await this.ready;
+    for (const id of ids) {
+      await this.deleteSession(id);
+    }
```

Comment:
I _think_ the redis package allows us to pass in an array of ids to delete. Don't quote me on that, though.